tfjs-backend-webgpu/src/adapter_info.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC. All Rights Reserved.
```

Comment:
According to https://github.com/tensorflow/tfjs/pull/6871#pullrequestreview-1124173265, please remove "All Rights Reserved." for the new files.